<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Builds</title>

<script src="site_libs/header-attrs-2.12/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Pierre BADY</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="builds.html">Builds</a>
</li>
<li>
  <a href="ressources.html">Ressources</a>
</li>
<li>
  <a href="publications.html">Publications</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Builds</h1>

</div>


<div id="r-package-modtools" class="section level1">
<h1><strong>R package modtools</strong></h1>
<p>Additional tools for model assessment modtools (alpha version): <a
href="https://github.com/badozor/modtools">modtools</a></p>
</div>
<div id="r-package-plstools" class="section level1">
<h1><strong>R package plstools</strong></h1>
<p>Partial Least Squares and Generalized Partial Least Squares models
based on NIPALS algorithm plstools (alpha version): <a
href="https://github.com/badozor/plstools">plstools</a></p>
</div>
<div id="r-package-ktabtools" class="section level1">
<h1><strong>R package ktabtools</strong></h1>
<p>Additional R functions related to K-table analyses ktabtools(in
construction). <a
href="https://github.com/badozor/ktabtools">ktabtools</a></p>
</div>
<div id="r-package-strtools" class="section level1">
<h1><strong>R package strtools</strong></h1>
<p>Tools for the analysis of STR fingerprinting data strtools (in
construction): <strong>strtools</strong></p>
</div>
<div id="r-package-methyltools" class="section level1">
<h1><strong>R package methyltools</strong></h1>
<p>Tools for the analysis of DNA methylation data (Infinium 27k and
450k) methyltools(alpha version). <a
href="https://github.com/badozor/methyltools">methyltools</a></p>
</div>
<div id="r-package-mgmtstp27" class="section level1">
<h1><strong>R package mgmtstp27</strong></h1>
<p>Prediction of DNA methylation state of MGMT promoter based on HM-450K
and HM-27K infinium platforms (from Illumina Company)</p>
<p><strong>Warning!</strong> The R package mgmtstp27 is still in
development and the documentation need some corrections (e.g. language,
etc …).</p>
<div id="installation" class="section level2">
<h2>Installation</h2>
<p>To install this package, load the archive containing the package
mgmtstp27 (see below), start R and enter:</p>
<pre class="r"><code>install.packages(&quot;mgmtstp27_0.2.tar.gz&quot;,repos=NULL)
install.packages(&quot;mgmtstp27_0.2.zip&quot;,repos=NULL)</code></pre>
</div>
<div id="documentation" class="section level2">
<h2>Documentation</h2>
<ul>
<li>Introduction to R package mgmtstp27 (version 0.1, in preparation):
intromgmtstp27.pdf</li>
<li>Effect of normalization on the prediction of DNA methylation status
of MGMT promoter: example with HM-450K * * Infinium data from TCGA and
the R package mgmtstp27 (version 0.1, in preparation):
docmgmtstp27.pdf</li>
<li>Prediction of the DNA methylation of MGMT with raw data (format
IDAT) from HM-27k (version 0.1, in preparation): MgmtPredTCGA.pdf</li>
</ul>
</div>
<div id="archives" class="section level2">
<h2>Archives</h2>
<p>Follow Installation instructions to use this package in your R
session</p>
<ul>
<li>Package Source: mgmtstp27_0.2.tar.gz (OLD version, see section
“depository” for new version)</li>
<li>Windows Binary: mgmtstp27_0.2.zip (32- &amp; 64-bit, OLD version,
see section “depository” for new version)</li>
<li>Browse/checkout source: log</li>
<li>Manual: mgmtstp27.pdf</li>
<li>Depository: <a
href="https://github.com/badozor/mgmtstp27">mgmtstp27</a></li>
</ul>
<p>Shiny application for R package mgmtstp27</p>
<ul>
<li>Depository: github</li>
</ul>
</div>
<div id="references" class="section level2">
<h2>References</h2>
<ul>
<li><p><strong>Bady, P.</strong>, D. Sciuscio, A.-C. Diserens, J. Bloch,
M. J. van den Bent, C. Marosi, P.-Y. Dietrich, M. Weller, L. Mariani, F.
L. Heppner, D. R. McDonald, D. Lacombe, R. Stupp, M. Delorenzi, and M.
E. Hegi. (2012). MGMT methylation analysis of glioblastoma on the
Infinium methylation BeadChip identifies two distinct CpG regions
associated with gene silencing and outcome, yielding a prediction model
for comparisons across datasets, tumor grades, and CIMP-status. Acta
Neuropathologica 124:547-560. PubMed</p></li>
<li><p><strong>Bady P.</strong>, Delorenzi M., Hegi M. (accepted)
Sensitivity analysis of the MGMT-STP27 model and impact of
genetic/epigenetic context to predict the MGMT methylation status in
gliomas and other tumors, Journal of Molecular Diagnostics, 18 (3),
350-361. PubMed</p></li>
</ul>
</div>
</div>
<div id="r-package-pbmisc" class="section level1">
<h1><strong>R package PBmisc</strong></h1>
<p>Contains many functions useful for statistical analyses, data
manipulation and graphic representations (<strong>in
construction</strong>): <a
href="https://github.com/badozor/PBmisc">PBmisc</a></p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
