<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Publications</title>

<script src="site_libs/header-attrs-2.27/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Pierre BADY</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="builds.html">
    <span class="fa fa-gears"></span>
     
    Builds
  </a>
</li>
<li>
  <a href="ressources.html">
    <span class="fa fa-toolbox"></span>
     
    Ressources
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-pdf"></span>
     
    Publications
  </a>
</li>
<li>
  <a href="misc.html">
    <span class="fa fa-paperclip"></span>
     
    Misc
  </a>
</li>
<li>
  <a href="https://github.com/badozor">
    <span class="fa fa-github"></span>
     
    GitHub
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Publications</h1>

</div>


<div id="articles" class="section level1">
<h1><strong>Articles</strong></h1>
<ol style="list-style-type: decimal">
<li><p>Gayraud S, Statzner B, Bady P, Haybachp A, Schöll F,
Usseglio-Polatera P, Bacchi M (2003). “Invertebrate traits for the
biomonitoring of large European rivers: an initial assessment of
alternative metrics.” <em>Freshwater Biology</em>, <em>48</em>(11),
2045-2064. <a href="doi:10.1046/j.1365-2427.2003.01139.x"
class="uri">doi:10.1046/j.1365-2427.2003.01139.x</a> <a
href="https://doi.org/10.1046/j.1365-2427.2003.01139.x"
class="uri">https://doi.org/10.1046/j.1365-2427.2003.01139.x</a>.</p></li>
<li><p>Bady P, Dolédec S, Dumont B, Fruget J (2004). “Multiple
co-inertia analysis: a tool for assessing synchrony in the temporal
variability of aquatic communities.” <em>Comptes rendus biologies</em>,
<em>327</em>(1), 29-36. <a href="doi:10.1016/j.crvi.2003.10.007"
class="uri">doi:10.1016/j.crvi.2003.10.007</a> <a
href="https://doi.org/10.1016/j.crvi.2003.10.007"
class="uri">https://doi.org/10.1016/j.crvi.2003.10.007</a>.</p></li>
<li><p>Statzner B, Bady P, Dolédec S, Schöll F (2005). “Invertebrate
traits for the biomonitoring of large European rivers: an initial
assessment of trait patterns in least impacted river reaches.”
<em>Freshwater Biology</em>, <em>50</em>(12), 2136-2161. <a
href="doi:10.1111/j.1365-2427.2005.01447.x"
class="uri">doi:10.1111/j.1365-2427.2005.01447.x</a> <a
href="https://doi.org/10.1111/j.1365-2427.2005.01447.x"
class="uri">https://doi.org/10.1111/j.1365-2427.2005.01447.x</a>.</p></li>
<li><p>Bady P, Dolédec S, Fesl C, Gayraud S, Bacchi M, Schöll F (2005).
“Use of invertebrate traits for the biomonitoring of European large
rivers: the effects of sampling effort on genus richness and functional
diversity.” <em>Freshwater Biology</em>, <em>50</em>(1), 159-173. <a
href="doi:10.1111/j.1365-2427.2004.01287.x"
class="uri">doi:10.1111/j.1365-2427.2004.01287.x</a> <a
href="https://doi.org/10.1111/j.1365-2427.2004.01287.x"
class="uri">https://doi.org/10.1111/j.1365-2427.2004.01287.x</a>.</p></li>
<li><p>Bigliardi-Qi M, Gaveriaux-Ruff C, Zhou H, Hell C, Bady P, Rufli
T, Kieffer B, Bigliardi PL (2006). “Deletion of <span
class="math inline">\(\delta\)</span>-opioid receptor in mice alters
skin differentiation and delays wound healing.”
<em>Differentiation</em>, <em>74</em>(4), 174-185. <a
href="doi:10.1111/j.1432-0436.2006.00065.x"
class="uri">doi:10.1111/j.1432-0436.2006.00065.x</a> <a
href="https://doi.org/10.1111/j.1432-0436.2006.00065.x"
class="uri">https://doi.org/10.1111/j.1432-0436.2006.00065.x</a>.</p></li>
<li><p>Matter M, Roulin D, Bady P, Liénard D, Gugerli O, Guggisberg D,
Cerottini J, Boubaker A, Alloua M, Lejeune F (2006). “105 POSTER
Sentinel lymph node (SLN) status is an important survival risk factor in
melanoma patients.” <em>European Journal of Surgical Oncology</em>,
S32-S33.</p></li>
<li><p>Matter M, Roulin D, Bady P, Liénard D, Voelter V, Gugerli O,
Guggisberg D, Cerottini J, Boubaker A, Lejeune F (2006). “Sentinel lymph
node (SLN) status is an important survival risk factor in melanoma
patients.” <em>Melanoma Research</em>, <em>16</em>, S56.</p></li>
<li><p>Mathez C, Bangala Y, Bady P, Zellweger J (2007). “Active
screening for pulmonary tuberculosis by chest x-ray among immigrants at
the Swiss border.” <em>Swiss Medical Weekly</em>, <em>137</em>(45-46),
649-654.</p></li>
<li><p>Daeppen J, Gaume J, Bady P, Yersin B, Calmes J, Givel J, Gmel G
(2007). “Brief alcohol intervention and alcohol assessment do not
influence alcohol use in injured patients treated in the emergency
department: a randomized controlled clinical trial.” <em>Addiction</em>,
<em>102</em>(8), 1224-1233. <a
href="doi:10.1111/j.1360-0443.2007.01869.x"
class="uri">doi:10.1111/j.1360-0443.2007.01869.x</a> <a
href="https://doi.org/10.1111/j.1360-0443.2007.01869.x"
class="uri">https://doi.org/10.1111/j.1360-0443.2007.01869.x</a>.</p></li>
<li><p>Daufresne M, Bady P, Fruget J (2007). “Impacts of global changes
and extreme hydroclimatic events on macroinvertebrate community
structures in the French Rhône River.” <em>Oecologia</em>,
<em>151</em>(3), 544-559. <a href="doi:10.1007/s00442-006-0655-1"
class="uri">doi:10.1007/s00442-006-0655-1</a> <a
href="https://doi.org/10.1007/s00442-006-0655-1"
class="uri">https://doi.org/10.1007/s00442-006-0655-1</a>.</p></li>
<li><p>Bigliardi-Qi M, Gaveriaux-Ruff C, Pfaltz K, Bady P, Baumann T,
Rufli T, Kieffer BL, Bigliardi PL (2007). “Deletion of <span
class="math inline">\(\mu\)</span>-and <span
class="math inline">\(\kappa\)</span>-opioid receptors in mice changes
epidermal hypertrophy, density of peripheral nerve endings, and itch
behavior.” <em>Journal of Investigative Dermatology</em>,
<em>127</em>(6), 1479-1488. <a href="doi:10.1038/sj.jid.5700661"
class="uri">doi:10.1038/sj.jid.5700661</a> <a
href="https://doi.org/10.1038/sj.jid.5700661"
class="uri">https://doi.org/10.1038/sj.jid.5700661</a>.</p></li>
<li><p>Fleming EA, Gmel G, Bady P, Yersin B, Givel J, Brown D, Daeppen J
(2007). “At-risk drinking and drug use among patients seeking care in an
emergency department.” <em>Journal of studies on alcohol and drugs</em>,
<em>68</em>(1), 28-35. <a href="doi:10.15288/jsad.2007.68.28"
class="uri">doi:10.15288/jsad.2007.68.28</a> <a
href="https://doi.org/10.15288/jsad.2007.68.28"
class="uri">https://doi.org/10.15288/jsad.2007.68.28</a>.</p></li>
<li><p>Lüthold S, Berneis K, Bady P, Müller B (2007). “Effects of
infectious disease on plasma lipids and their diagnostic significance in
critical illness.” <em>European Journal of Clinical Investigation</em>,
<em>37</em>(7), 573-579. <a href="doi:10.1111/j.1365-2362.2007.01826.x"
class="uri">doi:10.1111/j.1365-2362.2007.01826.x</a> <a
href="https://doi.org/10.1111/j.1365-2362.2007.01826.x"
class="uri">https://doi.org/10.1111/j.1365-2362.2007.01826.x</a>.</p></li>
<li><p>Roulin D, Matter M, Bady P, Liénard D, Gugerli O, Boubaker A,
Bron L, Lejeune F (2008). “Prognostic value of sentinel node biopsy in
327 prospective melanoma patients from a single institution.”
<em>European Journal of Surgical Oncology (EJSO)</em>, <em>34</em>(6),
673-679. <a href="doi:10.1016/j.ejso.2007.07.197"
class="uri">doi:10.1016/j.ejso.2007.07.197</a> <a
href="https://doi.org/10.1016/j.ejso.2007.07.197"
class="uri">https://doi.org/10.1016/j.ejso.2007.07.197</a>.</p></li>
<li><p>Indino P, Lemarchand P, Bady P, De Torrente A, Genné L, Genné D
(2008). “Prospective study on procalcitonin and other systemic infection
markers in patients with leukocytosis.” <em>International journal of
infectious diseases</em>, <em>12</em>(3), 319-324. <a
href="doi:10.1016/j.ijid.2007.09.007"
class="uri">doi:10.1016/j.ijid.2007.09.007</a> <a
href="https://doi.org/10.1016/j.ijid.2007.09.007"
class="uri">https://doi.org/10.1016/j.ijid.2007.09.007</a>.</p></li>
<li><p>Manuel O, Burnand B, Bady P, Kammerlander R, Vansantvoet M,
Francioli P, Zanetti G (2010). “Impact of standardised review of
intravenous antibiotic therapy 72 hours after prescription in two
internal medicine wards.” <em>Journal of Hospital Infection</em>,
<em>74</em>(4), 326-331. <a href="doi:10.1016/j.jhin.2009.07.011"
class="uri">doi:10.1016/j.jhin.2009.07.011</a> <a
href="https://doi.org/10.1016/j.jhin.2009.07.011"
class="uri">https://doi.org/10.1016/j.jhin.2009.07.011</a>.</p></li>
<li><p>Hegi ME, Diserens A, Bady P, Kamoshima Y, Kouwenhoven MC,
Delorenzi M, Lambiv WL, Hamou M, Matter MS, Koch A, others (2011).
“Pathway analysis of glioblastoma tissue after preoperative treatment
with the EGFR tyrosine kinase inhibitor gefitinib—a phase II trial.”
<em>Molecular cancer therapeutics</em>, <em>10</em>(6), 1102-1112. <a
href="doi:10.1158/1535-7163.mct-11-0048"
class="uri">doi:10.1158/1535-7163.mct-11-0048</a> <a
href="https://doi.org/10.1158/1535-7163.mct-11-0048"
class="uri">https://doi.org/10.1158/1535-7163.mct-11-0048</a>.</p></li>
<li><p>Hegi ME, Janzer R, Lambiv WL, Gorlia T, Kouwenhoven M, Hartmann
C, Von Deimling A, Martinet D, Besuchet Schmutz N, Diserens A, others
(2012). “Presence of an oligodendroglioma-like component in newly
diagnosed glioblastoma identifies a pathogenetically heterogeneous
subgroup and lacks prognostic value: central pathology review of the
EORTC_26981/NCIC_CE. 3 trial.” <em>Acta neuropathologica</em>,
<em>123</em>(6), 841-852.</p></li>
<li><p>Hegi M, Janzer R, Lambiv W, Gorlia T, Kouwenhoven M, Hartmann C,
von Deimling A, Martinet D, Besuchet Schmutz N, Diserens A, others
(2012). “European Organisation for Research and Treatment of Cancer
Brain Tumour and Radiation Oncology Groups; National Cancer Institute of
Canada Clinical Trials Group. Presence of an oligodendroglioma-like
component in newly diagnosed glioblastoma identifies a pathogenetically
heterogeneous subgroup and lacks prognostic value: central pathology
review of the EORTC_26981/NCIC_CE. 3 trial.” <em>Acta Neuropathol</em>,
<em>123</em>(6), 841-852.</p></li>
<li><p>Bady P, Diserens A, Castella V, Kalt S, Heinimann K, Hamou M,
Delorenzi M, Hegi ME (2012). “DNA fingerprinting of glioma cell lines
and considerations on similarity measurements.” <em>Neuro-oncology</em>,
<em>14</em>(6), 701-711. <a href="doi:10.1093/neuonc/nos072"
class="uri">doi:10.1093/neuonc/nos072</a> <a
href="https://doi.org/10.1093/neuonc/nos072"
class="uri">https://doi.org/10.1093/neuonc/nos072</a>.</p></li>
<li><p>Bady P, Sciuscio D, Diserens A, Bloch J, Van Den Bent MJ, Marosi
C, Dietrich P, Weller M, Mariani L, Heppner FL, others (2012). “MGMT
methylation analysis of glioblastoma on the Infinium methylation
BeadChip identifies two distinct CpG regions associated with gene
silencing and outcome, yielding a prediction model for comparisons
across datasets, tumor grades, and CIMP-status.” <em>Acta
neuropathologica</em>, <em>124</em>(4), 547-560. <a
href="doi:10.1007/s00401-012-1016-2"
class="uri">doi:10.1007/s00401-012-1016-2</a> <a
href="https://doi.org/10.1007/s00401-012-1016-2"
class="uri">https://doi.org/10.1007/s00401-012-1016-2</a>.</p></li>
<li><p>Logez M, Bady P, Pont D (2012). “Modelling the habitat
requirement of riverine fish species at the European scale: sensitivity
to temperature and precipitation and associated uncertainty.”
<em>Ecology of Freshwater Fish</em>, <em>21</em>(2), 266-282. <a
href="doi:10.1111/j.1600-0633.2011.00545.x"
class="uri">doi:10.1111/j.1600-0633.2011.00545.x</a> <a
href="https://doi.org/10.1111/j.1600-0633.2011.00545.x"
class="uri">https://doi.org/10.1111/j.1600-0633.2011.00545.x</a>.</p></li>
<li><p>Gutiérrez JS, González CA, Carrión MDB, de Jalón Lastra DG, de
Jalón González DG, Bady P, Pont D (2012). “Modelización paneuropea de
las comunidades piscícolas para la determinación del estado ambiental de
los ríos: aplicación a la cuenca del Tajo.” <em>Cuadernos de la Sociedad
Española de Ciencias Forestales</em>, 235-240.</p></li>
<li><p>Bady P, Sciuscio D, Diserens A, Bloch J, Van Den Bent MJ, Marosi
C, Dietrich P, Weller M, Mariani L, Heppner FL, others (2013). “Erratum
to: MGMT methylation analysis of glioblastoma on the Infinium
methylation BeadChip identifies two distinct CpG regions associated with
gene silencing and outcome, yielding a prediction model for comparisons
across datasets, tumor grades, and CIMP-status.” <em>Acta
Neuropathologica</em>, <em>126</em>(1), 159.</p></li>
<li><p>Logez M, Bady P, Melcher A, Pont D (2013). “A continental-scale
analysis of fish assemblage functional structure in European rivers.”
<em>Ecography</em>, <em>36</em>(1), 80-91. <a
href="doi:10.1111/j.1600-0587.2012.07447.x"
class="uri">doi:10.1111/j.1600-0587.2012.07447.x</a> <a
href="https://doi.org/10.1111/j.1600-0587.2012.07447.x"
class="uri">https://doi.org/10.1111/j.1600-0587.2012.07447.x</a>.</p></li>
<li><p>Kurscheid S, Sciuscio D, Bady P, Stupp R, Delorenzi M, Hegi M
(2014). “395: Epigenetic deregulation of HOXA genes associated with
aberrant expression in glioblastoma.” <em>European Journal of
Cancer</em>, S94.</p></li>
<li><p>Kurscheid S, Bady P, Sciuscio D, Samarzija I, Shay T, Vassallo I,
Van Criekinge W, Domany E, Stupp R, Delorenzi M, others (2014). “EG-05
COMBINATION OF GENE COPY GAIN AND EPIGENETIC DEREGULATION ARE ASSOCIATED
WITH THE ABERRANT EXPRESSION OF A STEM CELL RELATED HOX-SIGNATURE IN
GLIOBLASTOMA.” <em>Neuro-Oncology</em>, <em>16</em>(suppl_5),
v75-v76.</p></li>
<li><p>Kurscheid S, Bady P, Sciuscio D, Samarzija I, Shay T, Vassallo I,
Criekinge WV, Daniel RT, van den Bent MJ, Marosi C, others (2015).
“Chromosome 7 gain and DNA hypermethylation at the HOXA10 locus are
associated with expression of a stem cell related HOX-signature in
glioblastoma.” <em>Genome biology</em>, <em>16</em>(1), 1-15. <a
href="doi:10.1186/s13059-015-0583-7"
class="uri">doi:10.1186/s13059-015-0583-7</a> <a
href="https://doi.org/10.1186/s13059-015-0583-7"
class="uri">https://doi.org/10.1186/s13059-015-0583-7</a>.</p></li>
<li><p>De Meyer T, Bady P, Trooskens G, Kurscheid S, Bloch J, Kros JM,
Hainfellner JA, Stupp R, Delorenzi M, Hegi ME, others (2015).
“Genome-wide DNA methylation detection by MethylCap-seq and Infinium
HumanMethylation450 BeadChips: an independent large-scale comparison.”
<em>Scientific reports</em>, <em>5</em>(1), 1-13. <a
href="doi:10.1038/srep15375" class="uri">doi:10.1038/srep15375</a> <a
href="https://doi.org/10.1038/srep15375"
class="uri">https://doi.org/10.1038/srep15375</a>.</p></li>
<li><p>Sathyan P, Zinn PO, Marisetty AL, Liu B, Kamal MM, Singh SK, Bady
P, Lu L, Wani KM, Veo BL, others (2015). “Mir-21-Sox2 axis delineates
glioblastoma subtypes with prognostic impact.” <em>Journal of
Neuroscience</em>, <em>35</em>(45), 15097-15112. <a
href="doi:10.1523/jneurosci.1265-15.2015"
class="uri">doi:10.1523/jneurosci.1265-15.2015</a> <a
href="https://doi.org/10.1523/jneurosci.1265-15.2015"
class="uri">https://doi.org/10.1523/jneurosci.1265-15.2015</a>.</p></li>
<li><p>Weller M, Nabors LB, Gorlia T, Leske H, Rushing E, Bady P,
Hicking C, Perry J, Hong Y, Roth P, others (2016). “Cilengitide in newly
diagnosed glioblastoma: biomarker expression and outcome.”
<em>Oncotarget</em>, <em>7</em>(12), 15018-15032. <a
href="doi:10.18632\%2Foncotarget.7588"
class="uri">doi:10.18632\%2Foncotarget.7588</a> <a
href="https://doi.org/10.18632%5C%252Foncotarget.7588"
class="uri">https://doi.org/10.18632%5C%252Foncotarget.7588</a>.</p></li>
<li><p>Bady P, Delorenzi M, Hegi ME (2016). “Sensitivity analysis of the
MGMT-STP27 model and impact of genetic and epigenetic context to predict
the MGMT methylation status in gliomas and other tumors.” <em>The
Journal of molecular diagnostics</em>, <em>18</em>(3), 350-361. <a
href="doi:10.1016/j.jmoldx.2015.11.009"
class="uri">doi:10.1016/j.jmoldx.2015.11.009</a> <a
href="https://doi.org/10.1016/j.jmoldx.2015.11.009"
class="uri">https://doi.org/10.1016/j.jmoldx.2015.11.009</a>.</p></li>
<li><p>Wick W, Gorlia T, Bady P, Platten M, Van Den Bent MJ, Taphoorn
MJ, Steuve J, Brandes AA, Hamou M, Wick A, others (2016). “Phase II
study of radiotherapy and temsirolimus versus radiochemotherapy with
temozolomide in patients with newly diagnosed glioblastoma without MGMT
promoter hypermethylation (EORTC 26082).” <em>Clinical cancer
research</em>, <em>22</em>(19), 4797-4806. <a
href="doi:10.1158/1078-0432.ccr-15-3153"
class="uri">doi:10.1158/1078-0432.ccr-15-3153</a> <a
href="https://doi.org/10.1158/1078-0432.ccr-15-3153"
class="uri">https://doi.org/10.1158/1078-0432.ccr-15-3153</a>.</p></li>
<li><p>Oikonomaki M, Bady P, Hegi ME (2017). “Ubiquitin Specific
Peptidase 15 (USP15) suppresses glioblastoma cell growth via
stabilization of HECTD1 E3 ligase attenuating WNT pathway activity.”
<em>Oncotarget</em>, <em>8</em>(66), 110490-110502. <a
href="doi:10.18632/oncotarget.22798"
class="uri">doi:10.18632/oncotarget.22798</a> <a
href="https://doi.org/10.18632/oncotarget.22798"
class="uri">https://doi.org/10.18632/oncotarget.22798</a>.</p></li>
<li><p>Bady P, Kurscheid S, Delorenzi M, Gorlia T, van den Bent MJ,
Hoang-Xuan K, Vauléon É, Gijtenbeek A, Enting R, Thiessen B, others
(2018). “The DNA methylome of DDR genes and benefit from RT or TMZ in
IDH mutant low-grade glioma treated in EORTC 22033.” <em>Acta
neuropathologica</em>, <em>135</em>(4), 601-615. <a
href="doi:10.1007/s00401-018-1810-6"
class="uri">doi:10.1007/s00401-018-1810-6</a> <a
href="https://doi.org/10.1007/s00401-018-1810-6"
class="uri">https://doi.org/10.1007/s00401-018-1810-6</a>.</p></li>
<li><p>Hegi M, Cudalbu C, Bady P, Lai M, Xin L, Gusyatiner O, Hamou M,
Lepore M, Daniel R, Gruetter R, others (2018). “CBMT-15. METABOLIC AND
TRANSCRIPTIONAL PROFILES OF GBM INVASION: COMPARISON OF PATIENTS AND
PAIRED PATIENT DERIVED XENOGRAFTS USING1H MAGNETIC RESONANCE
SPECTROSCOPY AND IMAGING (7T AND 14T) AND RNA-SEQUENCING.”
<em>Neuro-Oncology</em>, <em>20</em>(suppl_6), vi35-vi35.</p></li>
<li><p>Gusyatiner O, Bady P, Pham MD, Lei Y, Park J, Daniel RT,
Delorenzi M, Hegi ME (2021). “BET inhibitors repress expression of
Interferon-stimulated genes and synergize with HDAC inhibitors in
glioblastoma.” <em>Neuro-oncology</em>, <em>23</em>(10), 1680-1692. <a
href="doi:10.1093/neuonc/noab115"
class="uri">doi:10.1093/neuonc/noab115</a> <a
href="https://doi.org/10.1093/neuonc/noab115"
class="uri">https://doi.org/10.1093/neuonc/noab115</a>.</p></li>
<li><p>Cudalbu C, Bady P, Lai M, Xin L, Gusyatiner O, Hamou M, Lepore M,
Brouland J, Daniel RT, Hottinger AF, others (2021). “Metabolic and
transcriptomic profiles of glioblastoma invasion revealed by comparisons
between patients and corresponding orthotopic xenografts in mice.”
<em>Acta Neuropathologica Communications</em>, <em>9</em>(1),
1-16.</p></li>
<li><p>Cudalbu C, Bady P, Lai M, Xin L, Gusyatiner O, Hamou M, Lepore M,
Brouland J, Daniel R, Hottinger A, others (2021). “OTME-13. Integration
of metabolic and transcriptional signatures of glioblastoma invasion
reveals extracellular matrix reorganization and vasculature remodeling.”
<em>Neuro-oncology Advances</em>, <em>3</em>(Suppl 2), ii16. <a
href="doi:10.1101/2021.06.29.450408"
class="uri">doi:10.1101/2021.06.29.450408</a> <a
href="https://doi.org/10.1101/2021.06.29.450408"
class="uri">https://doi.org/10.1101/2021.06.29.450408</a>.</p></li>
<li><p>Bady P, Marosi C, Weller M, Gronberg BH, Schultz H, Taphoorn MJ,
Gijtenbeek JM, van den Bent MJ, von Deimling A, Stupp R, others (2022).
“DNA methylation-based age acceleration observed in IDH wild-type
glioblastoma is associated with better outcome-including in elderly
patients.” <em>medRxiv</em>. <a href="doi:10.1101/2022.02.03.22270104"
class="uri">doi:10.1101/2022.02.03.22270104</a> <a
href="https://doi.org/10.1101/2022.02.03.22270104"
class="uri">https://doi.org/10.1101/2022.02.03.22270104</a>.</p></li>
<li><p>Darlix A, Bady P, Deverdun J, Lefort K, Rigau V, Le Bars E,
Meriadec J, Carrière M, Coget A, Santarius T, Matys T, Duffau H, Hegi ME
(2024). “Clinical value of the MGMT promoter methylation score in IDHmt
low-grade glioma for predicting benefit from temozolomide treatment.”
<em>Neuro-Oncology Advances</em>, <em>7</em>(1), vdae224. ISSN
2632-2498, <a href="doi:10.1093/noajnl/vdae224"
class="uri">doi:10.1093/noajnl/vdae224</a> <a
href="https://doi.org/10.1093/noajnl/vdae224"
class="uri">https://doi.org/10.1093/noajnl/vdae224</a>.</p></li>
</ol>
</div>
<div id="other-publications" class="section level1">
<h1><strong>Other publications</strong></h1>
<ol style="list-style-type: decimal">
<li><p>Bady P (2005). <em>Modélisation de la relation entre la
variabilité de l’habitat et la diversité fonctionnelle: application aux
peuplements de macroinvertébrés des grands cours d’eau européens</em>.
Ph.D. thesis, Lyon1.</p></li>
<li><p>Bady P, Dolédec S (2005). “Modelling and Prediction of Trait
Response and Functional Diversity of Stream Invertebrates: how to Deal
With Fuzzy Coded Species Traits?” In <em>AGU Spring Meeting
Abstracts</em>, volume 2005, NB13F-01.</p></li>
<li><p>Fruget J, Bady P, Olivier J, Carrel G, Souchon Y, Villeneuve B,
Daufresne M, Capra H (2006). <em>Etude thermique globale du Rhône phase
III, Lot 2: Etude à l’échelle du Rhône des compartiments
biologiques</em>. Ph.D. thesis, irstea.</p></li>
<li><p>Daeppen J, Gaume J, Bady P, Yersin B, Calmes J, Givel J, Gmel G
(2006). “Brief alcohol intervention for hazardous drinkers attending
emergency department: A randomized controlled trial.” In
<em>ALCOHOLISM-CLINICAL AND EXPERIMENTAL RESEARCH</em>, volume 30 number
6, 98A-98A. BLACKWELL PUBLISHING 9600 GARSINGTON RD, OXFORD OX4 2DQ,
OXON, ENGLAND.</p></li>
<li><p>Pont D, Bady P, Logez M (2008). “Uncertainties associated to
responses of fish bioindicator to environmental factors and human
disturbances in rivers. Consequences of global warming on good
ecological status.” In <em>EURAQUA Conference: How can climate change be
incorporated in river basin management plans under the
WFD?</em>.</p></li>
<li><p>Pont D, Bady P, Logez M, Veslot J (2009). <em>EFI+ Project.
Improvement and spatial extension of the European Fish Index Deliverable
4.1: Report on the modelling of reference conditions and on the
sensitivity of candidate metrics to anthropogenic pressures. Deliverable
4.2: Report on the final development and validation of the new European
Fish Index and method, including a complete technical description of the
new method. 6th Framework Programme Priority FP6-2005-SSP-5-A. N°
0044096. Rapport final</em>. Ph.D. thesis, irstea. <a
href="doi:10.13140/RG.2.2.35757.67042"
class="uri">doi:10.13140/RG.2.2.35757.67042</a> <a
href="https://doi.org/10.13140/RG.2.2.35757.67042"
class="uri">https://doi.org/10.13140/RG.2.2.35757.67042</a>.</p></li>
<li><p>Haidvogl G, Bady P, Pont D, Weiss M (2009). “The use of
historical data on diadromous fish to assess the effect of migration
barriers. Improving the ecological status of fish communities in inland
waters.” In <em>International Symposium and EFI+ Workshop</em>.</p></li>
<li><p>Pont D, Bady P, Logez M (2009). “The development of a new
predictive fish index at the European scale: 1. Model development and
metric selection. Improving the ecological status of fish communities in
inland waters.” In <em>International Symposium and EFI+ Workshop</em>,
6.</p></li>
<li><p>Bady P, Pont D, Logez M (2009). “Estimation of uncertainty and
predictive error for multimetric indices. Improving the ecological
status of fish communities in inland waters.” In <em>International
Symposium and EFI+ Workshop,</em>.</p></li>
<li><p>Logez M, Melcher A, Bady P, Pont D (2009). “Are biological and
ecological traits similarly linked in European fish assemblages?” In
<em>International Symposium and EFI+ Workshop, Hull, GBR, 30 March
2009</em>.</p></li>
<li><p>Logez M, Bady P, Pont D (2009). “Development of new metrics to
assess low species rivers, use of age and size classes of fish.
Improving the ecological status of fish communities in inland waters.”
In <em>International Symposium and EFI+ Workshop</em>.</p></li>
<li><p>Solana Gutierrez J, García de Jalón Lastra D, Pont D, Bady P,
Logez M, Noble R, Schinegger R, Haidvogl G, Melcher A, Schmutz S (2009).
“Manual for the application of the new European Fish Index-EFI+. A
fish-based method to assess the ecological status of European running
waters in support of the Water Framework Directive.” In
<em>International Symposium and EFI+ Workshop</em>.</p></li>
<li><p>Logez M, Pont D, Bady P (2011). “Modelling ecological niche of
fish species at the European scale: effects of climatic conditions and
potential consequencess on future species richness.” In <em>7th SEFS
Girona, ESP, 27 June-01 July 2011</em>, 15.</p></li>
<li><p>Bady P, Ehret G, Shamlaye C, Burnier M, Mach F, Paccaud F,
Delorenzi M, Bovet P, Bochud M (2012). “Genetic structure and admixture
of the Seychelles’ population.” In <em>20th Annual Conference of the
International Genetic Epidemiology Society (IGES 2011), Heidelberg,
Germany, September 18-20, 2011</em>, volume 36, 170-170.</p></li>
<li><p>Bady P, Sciuscio D, Stupp R, Delorenzi M, Hegi ME, others (2012).
“MGMT methylation based outcome prediction is associated with two CpG
regions separated by a prediction minimum centred at the initiation
start site.”</p></li>
<li><p>Bady P, Kurscheid S, Delorenzi M, Hegi ME (2013). “THE GENETIC
AND EPIGENETIC CONTEXT OF MGMT METHYLATION IN GLIOMA MAY IMPACT THE
PREDICTIVE AND PROGNOSTIC VALUE.” In <em>NEURO-ONCOLOGY</em>, volume 15,
141-142. OXFORD UNIV PRESS INC JOURNALS DEPT, 2001 EVANS RD, CARY, NC
27513 USA.</p></li>
<li><p>Hegi ME, Gorlia T, Bady P, Platten M, van den Bent MJ, Taphoorn
MJ, Steuve J, Brandes AA, Hamou M, Kosch M, others (2015). “Abstract
LB-A01: Molecular subgroup analysis of a randomized trial (EORTC
26082-22081) testing temsirolimus and radiation therapy versus
chemoradiotherapy with temozolomide in patients with newly diagnosed
glioblastoma without methylation of the MGMT gene promoter.”</p></li>
<li><p>Gusyatiner O, Pham MDT, Lei Y, Park J, Bady P, Delorenzi M, Hegi
M (2018). “BET inhibitors synergize with HDAC inhibitors and
downregulate expression of interferon response genes in
glioblastoma.”</p></li>
</ol>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
